pkgs/servers/nextcloud/packages/26.json

```diff
@@ -39,6 +39,16 @@
       "agpl"
     ]
   },
+  "files_markdown": {
```

Comment:
Is this a new package?

Reply:
It's an old app that was only recently made compatible with recent versions of Nextcloud: https://github.com/icewind1991/files_markdown/issues/200

Reply:
Got it